src/module.c

```diff
@@ -1249,7 +1249,7 @@ ValkeyModuleCommand *moduleCreateCommandProxy(struct ValkeyModule *module, sds d
  *                          the arguments that are channels.
  *
  * The last three parameters specify which arguments of the new command are
- * Redis keys. See https://redis.io/commands/command for more information.
+ * keys. See https://redis.io/commands/command for more information.
```

Comment:
Let's just change this too. I hope the URL will work in our new website.

```suggestion
 * keys. See https://valkey.io/commands/command for more information.
```